sdk/energyservices/Azure.ResourceManager.EnergyServices/Azure.ResourceManager.EnergyServices.sln

```diff
@@ -0,0 +1,51 @@
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
```

Comment:
Please run `eng\scripts\Update-Snippets.ps1 energyservices` to generate the api file to solve the ci issue

Reply:
I did. No files changed. 
 eng\scripts\Update-Snippets.ps1 energyservices
Tool 'azure.sdk.tools.snippetgenerator' is already installed.

SnippetGenerator completed in 00:00:02.2964892

Reply:
is there anything I missed?

Reply:
Ok, then sample is ok,
And need run `eng\scripts\Export-API.ps1 energyservices` to generate the api file, this is mandatory for ci pipeline